project/ISSUES.md

```diff
@@ -0,0 +1,179 @@
+  more information (or just more thinking) to proceed to a work in progress
+* `status/WIP` - this issue or PR is currently being worked on and in the case
+  of a PR, it should not be merged until this tag is removed
+* `status/review` - PR is complete and ready for review, or when applied to an
+  issue it is thought to be resolved but needs verification
+
+We use the same set of status tags for PRs and issues to keep things simple,
+but not every PR or issue needs to go through every state.  For example, it'll
+be common for a PR to be submitted with the label 'status/review', and get
+merged without needing to go through the rest of the states.
+
+Note that there are no `status/complete`, `status/wontfix`, `status/duplicate`,
+or other terminal status indicators.  This is intentional to keep the tail end
+of bookkeeping from getting onerous.  These sorts of terminal states *and their
+justifications* (e.g. the reason why it's a wontfix, or a reference to the
+duplicate issue) should be indicated in the closing comment by the issue
+closer.
+
+
+Area Labels
+------------
+
+The 'area' label is used for tagging issues and pull requests to a particular
+focus area.  This allows for easy searching within that particular domain, as
+well as more organized release notes.  Some examples, not-exhaustive, are here:
+
+* `area/API`
+* `area/cleanup`
+* `area/jobs`
+* `area/tests`
+* `area/GIEs`
+* `area/toolshed`
+* `area/UI-UX`
+* `area/workflows`
+
+This list will definitely grow over time.
+
+Other Useful Labels
+-------------------
+
+While the three labels sets indicating kind, status, and area are required
+there are several other labels that are be useful and/or have special purpose.
+
+* `procedures` is a special tag that indicates that the issue is related to
+  project governance, and it overrides the need for the trio of
+  kind/status/area tags, and these are never auto-flagged for triage.
+
+* `planning` is also a special tag that indicates the issue is related to
+  larger-scale issue planning.  These issues are typically meta-issues
+  containing checklists and references to other issues which are subcomponents
+  and stepping stones necessary for issue resolution.  These *can* utilize the
+  `focus/*` tags but are not required to.  Status and type make little sense
+  here.
+
+* `roadmap` is a reserved tag for the primary project roadmap.  This is a
+  meta-issue that is not expected to be completed, but rather serves as an
+  entry point to the high level development of the project.
+
+* `beginner-friendly` can be used to indicate a nice entry-level issue that
+  only requires limited understanding of the larger Galaxy framework and
+  ecosystem.  This is useful for encouraging new contributors.
+
+
+The Roadmap
+===========
+
+We will maintain a single `Roadmap` tagged meta-issue which will describe (at a
+very high level) the *current* major areas of focus for the project.  This is
```

Comment:
@jmchilton My goal is to find a process that works for all contributors. I was simply posing the question. I strongly agree it should be for the committers group at this stage, and would add that members of that group should be clear about their own motivations and priorities. My point regarding organizations has nothing to do with amount of contribution as much has process. As more organizations with more people get involved, how do we have a unified roadmap while also capturing organizational priorities? I'm not diminishing anyones contributions, I'm trying to be flexible in how different contributors contribute to the overall roadmap. (Is this a problem anyone has solved well?)